bin/xip-pdns

```diff
@@ -91,8 +93,8 @@ subdomain_is_ns() {
   [[ "$SUBDOMAIN" =~ $NS_SUBDOMAIN_PATTERN ]]
 }
 
-subdomain_is_ip() {
-  [[ "$SUBDOMAIN" =~ $IP_SUBDOMAIN_PATTERN ]]
```

Comment:
Do we still want to check the `IP_SUBDOMAIN_PATTERN` pattern?

Reply:
So like `[[ "$SUBDOMAIN" =~ $IP_SUBDOMAIN_PATTERN && "$SUBDOMAIN" =~ $LOCAL_IP_SUBDOMAIN_PATTERN ]]`?

Reply:
yep

Reply:
Done